4.9_alphabetical_order.py

```diff
@@ -0,0 +1,9 @@
+def alphabetical_order(words=''):
+    """takes an input form user in which the words seperated with hyphen icon(-), sorts the words in alphabetical order,
+    adds hyphen icon (-) between them and gives the output of it."""
+    words = words.split('-')
+    words.sort()
+    return '-'.join(words)
+
+
+print(alphabetical_order(input('Write the words with hyphen icon(-) between them: ')))
```

Comment:
kodlarınız çalışıyor tebrikler.
sadece diğer bazı çözümlerinizde de dikkatimi çeken fonksiyon parametrelerine  alphabetical_order(words='') gibi default değer atamanız. sadece parametreyi yazmanızda burdaki çözümler için yeterli olur.